src/map/utils/charutils.cpp

```diff
@@ -1358,9 +1358,12 @@ namespace charutils
             PChar->pushPacket(new CInventoryItemPacket(nullptr, LocationID, slotID));
             return 0;
         }
+
+        uint32 ItemID = PItem->getID();
```

Comment:
uint32 for an item ID? Isn't that kind of overkill

Reply:
I dunno, its what we already had. I just moved it up from line 1376